pkg/diagnose/connectivity/hooks.go

```diff
@@ -29,9 +26,12 @@ import (
 // During a request, the http.Client will call the functions of the ClientTrace at specific moments
 // This is useful to get extra information about what is happening and if there are errors during
 // connection establishment, DNS resolution or TLS handshake for instance
-func createDiagnoseTrace(writer io.Writer) *httptrace.ClientTrace {
+func createDiagnoseTraces(httpTraces []string) *httptrace.ClientTrace {
```

Comment:
The comment need to be updated to match the new function name.

Reply:
Thank you, I have updated the comment. Went through all other comments on the file and made some small adjustments as well.